front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -213,6 +217,27 @@ export function updateRowTimesAndMargin(
   return newRowData;
 }
 
+/**
+ * This function prevent a change from undefined to null (or the reverse)
+ * from being treated as an actual update of a row (otherwise would occur on deletion of an undefined field)
+ */
+export function preventUpdateOnNullableToNullable(
```

Comment:
Starting a stub discussion just so that we don't forget to bring this one to a conclusion before merging: https://github.com/OpenRailAssociation/osrd/pull/9739#issuecomment-2483556154